gitgud/levels/rampup/_reversing/instructions.txt

```diff
@@ -0,0 +1,19 @@
+Sometimes you may find that you made changes that you aren't happy with. Wouldn't it be great if there was a simple way to undo all your changes and return to the last commit? Here are two ways!
+>>>
+The commands "git reset" and "git revert" both allow you to rollback changes to an older commit. While "git reset" is used for the local branch, "git revert" is used for public branches.
```

Comment:
Can we add a brief explanation of what differentiates the two commands at a technical level?
git reset throws away the commit you're working on while git revert preserves the history while still undoing changes.

Also, instead of using the word public, can we change the description to this: `"git revert" is typically used for remote branches that other people are likely to have based their work off of.`

Reply:
Changed the "public" part to what you requested and added a brief explanation.